src/tests/posix/Mybuild

```diff
@@ -17,6 +17,11 @@ module all {
 	depends waitpid_test
 }
 
+module utime_test {
```

Comment:
Please, add this test to some template or to the `embox.test.posix.all` package to make sure it's used somewhere.

Reply:
added in tempate of x86/qemu

Reply:
set runlevel(2) for utime_test and its working

Reply:
last commit failed in travis, but it looks like an error in something not related to my commit, please see how will the time
"Error: Gold frame ./scripts/continuous/qt/gold_moveblocks_frames/3.jpg was not found in ./scripts/continuous/qt/captured.mpeg
- - - - - - - - - - - - - - -
ERROR: Test Qt moveblocks over vnc FAILED
- - - - - - - - - - - - - - -
error:
"